### PR TITLE
Heroic Query Testing

### PR DESCRIPTION
**Pull request by ryanvazquez**
Initial datasource testing for queries

### CODE REVIEW

File: .bumpedrc [hunk 1]
```diff
@@ -4,7 +4,7 @@ plugins:
   postrelease:
     Committing new version:
       plugin: bumped-terminal
-      command: 'git add package.json && git commit -m "Release $newVersion"'
+      command: 'git add -f -A package.json dist/ && git commit -m "Release $newVersion"'
```
Comment (dmichel1) [inline .bumpedrc]: something happened when you pulled these changes locally. but the build stuff should not be part of this PR

File: dist/types.d.ts [hunk 2]
```diff
@@ -0,0 +1,118 @@
+export interface RenderedQuery {
```
Comment (dmichel1) [inline dist/types.d.ts]: we should be able to stop checking in `dist` changes for now. the github releases will contain the `dist` folder that is created as part of `yarn build`

File: .bumpedrc (hunk 1 above)
Comment (ryanvazquez) [reply .bumpedrc]: Yep, not sure what happened there. I'll remove it.

File: dist/types.d.ts (hunk 2 above)
Comment (ryanvazquez) [reply dist/types.d.ts]: +1

Comment (ryanvazquez) [reply dist/types.d.ts]: +1
